packages/universal-components/src/RangeDatePicker/DayItemArrow.js

```diff
@@ -0,0 +1,99 @@
+// @flow
+
+import * as React from 'react';
+import { Animated, Easing, View, Platform } from 'react-native';
+
+import { type StylePropType, StyleSheet } from '../PlatformStyleSheet';
+import { designTokens } from '../DesignTokens';
+
+type Props = {|
+  +style?: StylePropType,
+|};
+
+type State = {|
+  scale: Animated.Value,
+|};
+
+const SHARED_ANIMATION_CONFIG = {
+  duration: designTokens.durationArrowScaleAnimation,
+  useNativeDriver: true,
+  easing: Easing.ease,
+};
+
+export default class DayItemArrow extends React.Component<Props, State> {
+  static animating: boolean;
```

Comment:
Does it need to be static?

Reply:
No, should I use it in state? And will be fine if I will set state in `componentDidMount`?

Reply:
Done